operations/patch_util.go

```diff
@@ -183,7 +183,15 @@ func (p *patchParams) validatePatchCommand(ctx context.Context, conf *ClientSett
 		grip.Warningf("warning - failed to set default alias: %v\n", err)
 	}
 
-	// Validate the alias if it exists
+	if err = p.loadVariants(conf); err != nil {
+		grip.Warningf("warning - failed to set default variants: %v\n", err)
+	}
+
+	if err = p.loadTasks(conf); err != nil {
+		grip.Warningf("warning - failed to set default tasks: %v\n", err)
```

Comment:
shouldn't we return here?

Reply:
I don't think this error should not stop us from going forward. 
The error is just in writing the default alias/variant/task to the user's config file.